warp-academy-docs/standards/atomic-asset.md

```diff
@@ -0,0 +1,121 @@
+# Introduction
+
+## What is atomic asset?
+
+An atomic asset is the concept, where **asset** (like an image) and **smart contract** are deployed atomically - in one logical transaction.
+It is possible, thanks to unique properties of arweave blockchain. In a typical blockchain asset is saved on some storage (for example IPFS) and only smart contract is deployed on-chain (for example ERC721). Why to choose atomic asset:
+
+- Standards like ERC-721 on Ethereum usually point to the external links of the NFT’s asset, and are stored outside of the Ethereum Virtual Machine, thus you have no guarantee that it will be stored there permanently. In scenario where storage provider is centralized you could wake up with empty NFT (pointing to non-existing URI) :( However even if u are using decentralized storage like IPFS, it doesn't provide strong guarantees about data availability like Arweave do. (you can read more [here](https://arweave.news/nfts-storage-arweave-vs-ipfs/)). In one example an artist named [neitherconfirm](https://twitter.com/neitherconfirm/status/1369285946198396928?s=20) changed the media files referenced by 26 of his NFTs to stock images of rugs. The artist later commented, “All discussions about the value of NFTs are meaningless as long as the token is not inseparable from the artwork its
```

Comment:
you could wake up empty-handed (NFT contract pointing to a non-existing URI). :(  However, even if you use a decentralized means of storage like IPFS, it doesn't offer the same level of assurance regarding data availability as Arweave.